test/test.py

```diff
@@ -152,6 +152,53 @@ def test_dl_transpose4(self):
         self.assertEqual(self._dl('bca', 'ab'), 2.5)
 
 
+class TestClev_noinit(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def _lev(self, x, y):
+        return lev(x, y)
+
+    def _osa(self, x, y):
+        return osa(x, y)
+
+    def _dl(self, x, y):
+        return dam_lev(x, y)
+
+    def test_lev_noinit(self):
```

Comment:
If we rename the class above, don't forget to rename this test to correspond with it. It would also be acceptable to just rename this test method to `test_lev`. Same goes with the remaining test methods.